### PR TITLE
feat(bin/emqx): add `-init_debug` system arg when `DEBUG=2`

### PR DESCRIPTION
**Pull request by thalesmg**
Fixes <issue-or-jira-number>

Release version: v/e5.8.0

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Added property-based tests for code which performs user input validation
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: bin/emqx [hunk 1]
```diff
@@ -330,7 +330,7 @@ COMPATIBILITY_CHECK='
     end,
     halt(0).
 '
-[ "$DEBUG" -eq 1 ] && set -x
+[[ "$DEBUG" -gt 1 ]] && set -x
```
Comment (zmstone) [inline bin/emqx:333]:
```suggestion
[[ "$DEBUG" -gt 0 ]] && set -x
```

File: bin/emqx [hunk 2]
```diff
@@ -558,7 +558,7 @@ else
     fi
 fi
 logdebug "EMQX_BOOT_CONFIGS: $EMQX_BOOT_CONFIGS"
-[ "$DEBUG" -eq 1 ] && set -x
+[[ "$DEBUG" -gt 1 ]] && set -x
```
Comment (zmstone) [inline bin/emqx:561]:
```suggestion
[[ "$DEBUG" -gt 0 ]] && set -x
```

File: bin/emqx [hunk 3]
```diff
@@ -1239,6 +1239,12 @@ case "${COMMAND}" in
         # Build an array of arguments to pass to exec later on
         # Build it here because this command will be used for logging.
         if [ "$IS_ELIXIR" = no ] || [ "${EMQX_CONSOLE_FLAVOR:-}" = 'erl' ] ; then
+            if [[ "$DEBUG" == 2 ]]; then
```
Comment (zmstone) [inline bin/emqx:1242]:
```suggestion
            if [[ "$DEBUG" -eq 2 ]]; then
```

File: bin/emqx [hunk 4]
```diff
@@ -1249,8 +1255,15 @@ case "${COMMAND}" in
                 -mode "$CODE_LOADING_MODE" \
                 -config "$CONF_FILE" \
                 -args_file "$ARGS_FILE" \
+                $INIT_DEBUG_ARG \
                 $EPMD_ARGS
         else
+            if [[ "$DEBUG" == 2 ]]; then
```
Comment (zmstone) [inline bin/emqx:1261]:
```suggestion
            if [[ "$DEBUG" -eq 2 ]]; then
```

File: bin/emqx (hunk 3 above)
Comment (thalesmg) [reply bin/emqx:1242]: Isn't it the same thing?

**Review by zmstone — APPROVED** [review]

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-3445)

**Pull request merged**